CHANGELOG.md

```diff
@@ -12,15 +12,21 @@
 
 ### Changed
 
+- [BREAKING] `Overlay`: The Overlay component should contain children instead of being positioned adjacent to children ([@ArnaudWeyts](https://github.com/ArnaudWeyts) in [#1729](https://github.com/teamleadercrm/ui/pull/1729))
```

Comment:
I guess we should also mention the removal of these props
```
onOverlayMouseDown
onOverlayMouseMove
onOverlayMouseUp,
```

Reply:
Good point!